### PR TITLE
refactor(v3): VuetifyLoaderPlugin

### PR DESCRIPTION
**Pull request by ElijahKotyluk**

### CODE REVIEW

File: packages/vue-cli-plugin-vuetify/index.js [hunk 1]
```diff
@@ -12,15 +12,18 @@ module.exports = (api) => {
...
   if (hasVuetifyLoader) {
-    const vueVersion = semver.major(require('vue/package.json').version)
-    const VuetifyLoaderPlugin = vueVersion === 3 ? require('vuetify-loader').VuetifyLoaderPlugin : require('vuetify-loader/lib/plugin')
+    const VuetifyLoaderPlugin = require('vuetify-loader').VuetifyLoaderPlugin
 
-    api.chainWebpack(config => {
-      config
-        .plugin('VuetifyLoaderPlugin')
-        .use(VuetifyLoaderPlugin)
-    })
```
Comment (KaelWD) [inline packages/vue-cli-plugin-vuetify/index.js:23]: This is still needed for v2

File: packages/vue-cli-plugin-vuetify/generator/templates/v3/src/plugins/vuetify.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 // Styles
 import '@mdi/font/css/materialdesignicons.css'
+<%_ if (useVite) { _%>
```
Comment (KaelWD) [inline packages/vue-cli-plugin-vuetify/generator/templates/v3/src/plugins/vuetify.js:3]: This doesn't have anything to do with vite, it's the same for both vite and webpack.

File: packages/vue-cli-plugin-vuetify/generator/tools/alaCarte.js [hunk 3]
```diff
@@ -16,6 +16,18 @@ function addDependencies (api, useV3) {
   api.extendPackage(deps)
 }
 
+function addVueConfigPluginOptions(api) {
+  api.extendPackage({
+    vue: {
+      pluginOptions: {
+        vuetify: {
+          styles: 'expose',
```
Comment (KaelWD) [inline packages/vue-cli-plugin-vuetify/generator/tools/alaCarte.js:24]: Do we want this to be the default?

Comment (johnleider) [reply packages/vue-cli-plugin-vuetify/generator/tools/alaCarte.js:24]: No, the default should be to load the component's pre-compiled CSS file.

File: packages/vue-cli-plugin-vuetify/generator/tools/alaCarte.js [hunk 4]
```diff
@@ -26,7 +38,11 @@ function addVueConfigTranspileDependency (api) {
...
 }
 
+function addVueConfigVuetify(api, useV3) {
+  useV3 ? addVueConfigPluginOptions(api) : addVueConfigTranspileDependency(api)
+}
+
 module.exports = {
   addDependencies,
-  addVueConfigTranspileDependency,
+  addVueConfigVuetify
```
Comment (johnleider) [inline packages/vue-cli-plugin-vuetify/generator/tools/alaCarte.js:47]:
```suggestion
  addVueConfigVuetify,
```

**Review by johnleider — CHANGES_REQUESTED** [review]

File: packages/vue-cli-plugin-vuetify/generator/tools/alaCarte.js [hunk 5]
```diff
@@ -20,9 +20,7 @@ function addVueConfigPluginOptions(api) {
   api.extendPackage({
     vue: {
       pluginOptions: {
-        vuetify: {
-          styles: 'expose',
-        },
+        vuetify: {},
```
Comment (johnleider) [inline packages/vue-cli-plugin-vuetify/generator/tools/alaCarte.js:23]: ``
```suggestion
        vuetify: {
          // https://github.com/vuetifyjs/vuetify-loader/tree/next/packages/vite-plugin
        },
```

**Review by johnleider — CHANGES_REQUESTED** [review]

File: packages/vue-cli-plugin-vuetify/generator/index.js [hunk 6]
```diff
@@ -1,4 +1,6 @@
 module.exports = (api, opts) => {
+  const { fileExists } = require('../util/helpers')
+
```
Comment (johnleider) [inline packages/vue-cli-plugin-vuetify/generator/index.js:3]:
```suggestion
```

**Review by johnleider — APPROVED** [review]

**Pull request merged**